src/style.css

```diff
@@ -0,0 +1,193 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: linear-gradient(120deg, #665521, #3d2b26);
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  min-height: 100vh;
+}
+
+header {
+  font-size: 1.5rem;
+}
+
+header,
+form {
+  min-height: 20vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+form input,
+form button {
+  padding: 0.5rem;
+  font-size: 2rem;
+  border: none;
+  background: #fff;
+}
+
+form button {
+  color: #3d2b26;
+  background: #fff;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+form button:hover {
+  background: #3d2b26;
+  color: #fff;
+}
+
+.todo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.todo-list {
+  min-width: 30%;
+  list-style: none;
+}
+
+.todo {
+  margin: 0.5rem;
+  background: #fff;
+  color: #000;
+  font-size: 1.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: all 0.3s ease;
+}
+
+.todo li {
+  flex: 1;
+}
+
+.trash-btn,
+.complete-btn {
+  background: #b32323;
+  color: #fff;
+  border: none;
+  margin: 5px 2px  0 5px;
+  padding: 5px;
+  cursor: pointer;
+  font-size: 1rem;
+}
+
+.edit-btn {
+  background: #227;
+  color: #fff8db;
+  border: none;
+  margin: 5px 0  2px 5px;
+  padding: 5px;
+  cursor: pointer;
+  font-size: 1rem;
+}
+
+.complete-btn {
+  background: #97be6f;
+  margin: 5px 5px  2px 5px;
+  padding: 5px;
+  color: #000;
+}
+
+.todo-item {
+  padding: 0 0.5rem;
+}
+
+.fa-trash {
+  pointer-events: none;
+}
+
+.fa-check {
+  pointer-events: none;
+  color: #fff;
+}
+
+.completed {
+  text-decoration: line-through;
+  opacity: 0.5;
+}
+
+.fall {
+  transform: translateY(8rem) rotateZ(20deg);
+  opacity: 0;
+}
+
+select {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  outline: none;
+  border: none;
+  color: black;
+  width: 10rem;
+  cursor: pointer;
+  padding: 1.1rem;
+}
+
+.select {
+  margin: 1.1rem;
+  position: relative;
+  overflow: hidden;
+}
+
+.select::after {
+  content: "\25BC";
+  position: absolute;
+  background: #3d2b26;
+  top: 0;
+  right: 0;
+  padding: 1.1rem;
+  pointer-events: none;
+  transition: all 0.3s ease;
+}
+
+.select:hover::after {
+  background: white;
+  color: #3d2b26;
+}
+
+.alert-success {
+  font-family: Verdana, sans-serif;
+  font-size: 15px;
+  line-height: 1rem;
+  width: 30vw;
+  box-sizing: inherit;
+  transition: opacity 0.15s linear;
+  padding: 5px;
```

Comment:
you used padding: 5px more than 5 times, so it will be better if you can create a variable for this padding

Reply:
- Noted! I will address such a concern in all honesty. 🤗